Xamarin.Forms.Core/Items/CarouselView.cs

```diff
@@ -168,6 +168,38 @@ public LinearItemsLayout ItemsLayout
 			set => SetValue(ItemsLayoutProperty, value);
 		}
 
+
+		[TypeConverter(typeof(ReferenceTypeConverter))]
+		public IndicatorView IndicatorView
```

Comment:
does this need to be a BP? because it'll keep a ref to the indicatorview.

Reply:
Fixed